db/migrate/20171210195325_create_tenants.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTenants < ActiveRecord::Migration[5.1]
+  def change
+    create_table :tenants do |t|
+      t.string :name
```

Comment:
tenant belongs to user. add this relation
t.integer :owner_id, index: true

class Tenant
  has_one :owner, class_name: User

Reply:
done